In [4]:
""" Complete the following code to find the area of an equilateral triangle. Output should be as displayed """
from math import sqrt
try:
  side = float(input("Enter the side of the equilateral triangle: "))
  area = round(((sqrt(3))/4)*pow(side,2),2)
  print("Area of the equilateral triangle with side",side,"is:",area)
except:
  print("Expecting a numeric value!")

Enter the side of the equilateral triangle: 3
Area of the equilateral triangle with side 3.0 is: 3.9


Output:
Enter the side of the equilateral triangle: 3
Area of the equilateral triangle with side 3.0 is: 3.9